packages/x-charts-pro/src/FunnelChart/funnel.types.ts

```diff
@@ -43,7 +43,7 @@ export interface FunnelSeriesType
   /**
    * Data associated to the funnel section.
    */
-  data?: Readonly<FunnelValueType[]>;
+  data: Readonly<FunnelValueType[]>;
```

Comment:
Just noticed the Type is not the same as pie chart.

We should probably either keep that, or make the `pie.data` optional

Reply:
What do you mean?

Reply:
I noticed pie series `data` attribute was required, and the one of funnel optional. But could not decide which approach the correct one.

I now remember that the reason why other series type have `data` as an optional param, is because they also support `dataKey`.

Since neither pie or funnel support  `dataKey` the `data`  should be defined.